core/src/consensus.rs

```diff
@@ -1425,6 +1456,7 @@ pub mod test {
                 &mut self.heaviest_subtree_fork_choice,
                 &mut true,
                 &mut Vec::new(),
+                &mut BTreeMap::new(),
```

Comment:
This new `BTreeMap` friend comes after `bool` and `Vec` friends now.  Plumbed like this throughout